pages/platform/ai/notebook_tuto_marine_mammal_sounds_classification/guide.en-gb.md

```diff
@@ -0,0 +1,88 @@
+---
+title: AI Notebooks - Tutorial - Audios analyse and classification with AI
+slug: notebooks/tuto-marine-mammal-sounds-classification
+excerpt: How to classify sounds with AI
+section: AI Notebooks tutorials
+order: 6
+---
+
+**Last updated 8th of February, 2022.**
+
+
+## Objective
+
+The purpose of this tutorial is to show how it is possible to train a model in oder to classify sounds. To do this, we take as an example a dataset of **marine mammals sounds**.
+
+![image](images/marine-mammals-categories.png){.thumbnail}
+
+## Requirements
+
+* access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager)
+* an AI Notebooks project created inside a Public Cloud project
+* a user for AI Notebooks
+* your own dataset
+
+## Instructions
+
+### Uploading your dataset on Public Cloud Storage
+
+If you want to upload it from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager), go to the Object Storage section and create a new object container by clicking `Object Storage` > `Create an object container`.
+
+![image](images/new-object-container.png){.thumbnail}
+
+If you want to run it with the CLI, just follow this [guide][OVH DATA cli]. You have to choose the region, the name of your container and the path where your data is located and use the following command:
+
+```bash
+ovhai data upload <region> <container> <paths>
+```
+
+>[!primary]
+>
+> This tutorial has been realized with the [Best of Watkins Marine Mammal Sound Database](https://cis.whoi.edu/science/B/whalesounds/index.cfm). If you don't have your own dataset, you can use it by downloading the dataset on [Kaggle](https://www.kaggle.com/shreyj1729/best-of-watkins-marine-mammal-sound-database/version/3).
+>
+
+### Launching and accessing Jupyter notebook with "One image to rule them all" framework
+
+>[!warning]
+>
+> Although this tutorial is based on the use of the **TensorFlow** image, we advise you to use this image: **One image to rule them all**. This will help you avoid errors when installing libraries such as Librosa or SoundFile, Python audio libraries.
+>
+
+You need to attach a volume if your data is in your OVHcloud Object Storage and you want to use it during your experiment. For more information on data, volumes and permissions, see [our guide on data][OVH Data].
+
+If you want to launch it from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager), just follow this [guide][OVH Notebooks].
```

Comment:
Missing link. Please edit it and replace [Text for link][URL] with [Text for link](URL).